portingdb/templates/index.html

```diff
@@ -154,9 +157,16 @@ <h2>Package Groups</h2>
                             >
                             {{ grp.name }}
                         </a>
+                        {% if 'exception' in grp %}
+                            <small>(<a href="{{ grp['exception'] }}">exc.</a>)</small>
+                        {% endif %}
```

Comment:
Suggestion: Else: (needs exc.)

Reply:
Not necessarily; portingdb might be outdated. I think the warning above is enough.